src/App.vue

```diff
@@ -37,71 +72,112 @@ export default {
   data() {
     return {
       showModal: false,
-    }
+    };
   },
   watch: {
     theme(value) {
-      document.documentElement.classList.remove('dark', 'light', 'october')
-
-      switch(value) {
-        case 'dark':
-          document.documentElement.classList.add('dark')
-          break
-        case 'light':
-          document.documentElement.classList.add('light')
-          break
-        case 'october':
-          document.documentElement.classList.add('dark', 'october')
-          break
+      document.documentElement.classList.remove("dark", "light", "october");
+
+      switch (value) {
+        case "dark":
+          document.documentElement.classList.add("dark");
+          break;
+        case "light":
+          document.documentElement.classList.add("light");
+          break;
+        case "october":
+          document.documentElement.classList.add("dark", "october");
+          break;
+        case "auto dark":
+          document.documentElement.classList.add("auto dark");
+          break;
+        case "auto light":
+          document.documentElement.classList.add("auto light");
+
         default:
-          document.documentElement.classList.add('dark')
-          break
+          document.documentElement.classList.add("dark");
+          break;
       }
     },
+    updateTheme(){
```

Comment:
This is super close, but `updateTheme` needs to be moved down inside the `methods: {}` block below. In Vue, everything inside `methods` is a function that can be called anywhere else inside this component.